.github/workflows/reusable_release_tracking_epic.yml

```diff
@@ -0,0 +1,117 @@
+        description: "Which repo is triggering this release?"
+      release-version:
+        required: true
+        type: string
+        description: "What is the version triggering this release?"
+
+jobs:
+  create_issue:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: dacbd/create-issue-action@v2
+        id: create-issue
+        with:
+          # This is an org level secret. No changes needed at the repo level
+          token: ${{ secrets.CREATE_RELEASE_EPIC }}
+          owner: celestiaorg
+          # Creating the tracking issue in devops as devops is responsible for
+          # updating the networks
+          repo: devops
+          # Currently unable to assign a group to an issue, so assigning MSevey
+          # as issue owner.
+          assignees: MSevey
+          # Title and body and free to change based on the team's needs
+          title: ${{inputs.release-repo}} ${{inputs.release-version}} Release Deployment Tracking
+          body: |
+            ## Release Checklist
+            ```[tasklist]
+            ### Pre-work
+            - [ ] Confirm release tested on Robusta
+            - [ ] App and Node compatibility confirmed
+            - [ ] Release notes reviewed
+            ```
+            ```[tasklist]
+            ### Stage 1: Arabica
+            - [ ] Prepare deployment to Arabica
+            - [ ] Prepare announcement for Arabica
+            - [ ] Announce release for Arabica
+            - [ ] Deploy to Arabica
+            - [ ] Verify release on Arabica
+            - [ ] Docs updated for release for Arabica
+            ```
+            ```[tasklist]
+            ### Stage 2: Mocha
+            - [ ] Prepare deployment to Mocha
+            - [ ] Prepare announcement for Mocha
+            - [ ] Announce release for Mocha
+            - [ ] Deploy to Mocha
+            - [ ] Verify release on Mocha
+            - [ ] Docs updated for release for Mocha
+            ```
+            ```[tasklist]
+            ### Stage 3: Mainnet
+            - [ ] Prepare deployment for mainnet
+            - [ ] Prepare announcement for mainnet
+            - [ ] Announce release for mainnet
+            - [ ] Deploy to mainnet
+            - [ ] Verify release on mainnet
+            - [ ] Docs updated for release for mainnet
+            ```
+            cc @celestiaorg/devops
+
+      - uses: ravsamhq/notify-slack-action@v2
+        with:
+          status: ${{ job.status }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          notification_title: "New Release for Deployment"
+          message_format: "{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}>"
+          footer: "Linked Issue <${{steps.create-issue.outputs.html_url}}>"
+          # This is the devops user group
+          mention_groups: "S04MZ965WQZ"
+        env:
+          # This is an org level secret and is currently set to #releases in
+          # slack
```

Comment:
The reason I was thinking keep it in the release channel was because one of the first steps I would take would be to look at the release, and so it would be nice for this notification to be right next to the release notification from the repo. 
Since I only expect this to apply to app and node, I would only expect maybe 1 ping a month based on how frequently the teams make releases. 
So having a totally separate channel seems unnecessary imo.

Reply:
Works for me. Thanks for explaining.